sherpa/sherpa-standalone.rc

```diff
@@ -1,7 +1,7 @@
 # SHERPA_VERSION 4.14.1
```

Comment:
This should probably get changed to `4.14.2` but I don't know what the rule we have been using for this (it is purely decorative).